examples/annotation_import/text.ipynb

```diff
@@ -39,9 +39,10 @@
         "* Classification radio \n",
```

Comment:
<p>Could you please uncomment this code? since we don't use relationships we can now use Label import. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>